Submitted By:     VMiklos <mamajom@axelero.hu>
Date:             2005.01.21
Initial Packages: 2.9.4
Description:      This patch adds a new feature to makepkg.
                  Automatically update the last modified line in FrugalBuild
                  after a successfully build.

diff -Naur pacman-2.9.4.orig/scripts/makepkg pacman-2.9.4/scripts/makepkg
--- pacman-2.9.4.orig/scripts/makepkg   2005-01-07 00:53:27.000000000 +0100
+++ pacman-2.9.4/scripts/makepkg        2005-01-07 00:56:05.051397736 +0100
@@ -713,6 +713,9 @@
        pacman -R $makedeplist $deplist
 fi

+# update last mod line
+sed -i "s/^\(# Last modified: \).*/\1`date -R`/i" $BUILDSCRIPT
+
 msg "Finished making: $pkgname  (`date`)"

 if [ "$INSTALL" = "1" -a "`id -u`" = "0" -a "$INFAKEROOT" != "1" ]; then
